server.js

```diff
@@ -2,23 +2,28 @@
 
 const IPFS = require('ipfs')
 const OrbitDB = require('orbit-db')
+const Pubsub = require('orbit-db-pubsub')
 const Room = require('ipfs-pubsub-room')
 
 const ipfsOptions = {
   EXPERIMENTAL: {
     pubsub: true
   }
 }
+const ORBITDB_PATH = './orbitdb' //'/opt/orbitdb'
```

Comment:
This is a fixed path on the server, not supposed to change.

Reply:
Oh, sorry. I was using this to test locally. will update!